src/index.html

```diff
@@ -22,6 +22,362 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <div class="container">
+        <div class="header-img">
+          <a
+            href="#"
+            class="logo-img"
+          >
+            <img
+              src="images/logo.png"
+              alt="logo"
+            />
+          </a>
+        </div>
+        <nav class="nav">
+          <ul class="nav__list">
+            <li class="nav__item">
+              <a
+                href="#"
+                class="nav__link is-active"
+              >
+                Apple
+              </a>
+            </li>
+            <li class="nav__item">
```

Comment:
Add the blank line between the links
```suggestion
            </li>
            
            <li class="nav__item">
```